packages/manager/modules/vrack/src/dashboard/translations/Messages_fr_FR.json

```diff
@@ -1,7 +1,9 @@
 {
   "vrack_introduction_title": "Composer votre vRack",
   "vrack_introduction_description": "Gérez l'interconnexion privée de tous les composants de votre infrastructure informatique, partout dans le monde. <br/> La technologie vRack (baie virtuelle) permet de connecter, isoler ou répartir vos services au sein d'un ou plusieurs réseau privés et sécurisés.",
-  "vrack_introduction_change_owner": "Vous avez aussi la possibilité de <a href=\"{{ uri }}\" target=\"_blank\">changer de propriétaire</a> si vous le désirez.",
+  "vrack_introduction_change_owner": "Vous avez aussi la possibilité de",
```

Comment:
Better keep initial translation because here you cut the sentence and it can be hard to translate on other languages